templates/listcollectorcontent.html

```diff
@@ -0,0 +1 @@
+{% include 'question.html' %}
```

Comment:
The logic in question was updated to handle content. Just call `list-content.html` here?.

Reply:
Upated!